servers/Fabric.Mcp.Server/vscode/package.json

```diff
@@ -0,0 +1,130 @@
+{
+    "name": "vscode-fabric-mcp-server",
+    "displayName": "Microsoft Fabric MCP Server",
+    "description": "Provides Model Context Protocol (MCP) integration and tooling for Fabric in Visual Studio Code.",
+    "version": "0.0.0",
+    "license": "MIT",
+    "icon": "resources/package-icon.png",
+    "preview": false,
+    "publisher": "ms-fabrictools",
```

Comment:
This doesn't look right for the publisher. 
the current fabric publisher ID on VS code marketpace is fabric.